cabal.project

```diff
@@ -66,7 +115,9 @@ constraints:
   ghc-check -ghc-check-use-package-abis,
   ghc-lib-parser-ex -auto,
   stylish-haskell +ghc-lib,
-  fourmolu -fixity-th
+  fourmolu -fixity-th,
+  setup.happy == 1.20.1.1,
```

Comment:
why aren't these constraints in the conditional block? comments? can we put the 9.4 constraints in a conditional block also?